sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared/shared_access_signature.py

```diff
@@ -39,6 +39,8 @@ class QueryStringConstants(object):
     SIGNED_KEY_EXPIRY = 'ske'
     SIGNED_KEY_SERVICE = 'sks'
     SIGNED_KEY_VERSION = 'skv'
+
+    # for blob only
```

Comment:
These comments are actually wrong now as its for blob and datalake :). If you wanted, you could remove them and then make blob align but this is optional for this change I would say.